test/dynamo/test_flash_attention.py

```diff
@@ -0,0 +1,43 @@
+# Owner(s): ["module: dynamo"]
+import contextlib
+
+import torch._dynamo.test_case
+import torch._dynamo.testing
+from torch.backends.cuda import SDPAParams
+
+
+@contextlib.contextmanager
+def allow_in_graph_sdpa_params():
+    global SDPAParams
+    try:
+        old = SDPAParams
+        SDPAParams = torch._dynamo.allow_in_graph(SDPAParams)
+        yield
+    finally:
+        SDPAParams = old
+
+
+class TestFlashAttention(torch._dynamo.test_case.TestCase):
+    def test_returns_SDPAParams(self):
```

Comment:
In this test, we take a set of tensors and produce the object as an intermediary, and return it.

Please add a test that:

1) Shows this as input
2) Shows it being created on one side of a graph break, and used on the other side (to prove reconstruct works)